Orso.Arpa.Infrastructure/Authentication/JwtGenerator.cs

```diff
@@ -119,5 +75,25 @@ private RefreshToken GernerateRefreshToken(User user, string ipAddress)
                 now
             );
         }
+
+        public async Task<ClaimsIdentity> GetClaimsIdentity(User user)
+        {
+            var claims = new List<Claim>
+            {
+                new Claim(JwtRegisteredClaimNames.NameId, user.UserName),
+                new Claim(JwtRegisteredClaimNames.Name, user.DisplayName),
```

Comment:
we shouldn't put the display name into the token anymore as it is a personal data.